### ISSUE TITLE
Show negative timer values

### ISSUE THREAD

**Issue by struesda** [kzhovn/statusbar-pomo-obsidian#19]
If a timer runs out - but is not paused, can the timer display show a negative number?  
That way I can tell how FAR past the timer I am.

**Comment by kzhovn** [comment]
Not sure what you're visualizing here - right now, the timer either goes straight to a break, or pauses automatically, depending on whether you've got autostart enabled. Do you want a third setting that, instead of pausing or continuing on to a break at the end of the timer, goes into the negatives until you pause it manually?

**Comment by struesda** [comment]
yes that is what I had in mind.  This is handy for those times when I am tempted to get that "one extra minute" in to finish something - but when that turns into "10 extra minutes" - a negative timer helps me to spot when I am slipping into that!
